docs/concepts/work-pools.md

```diff
@@ -154,9 +154,16 @@ Inspect the work pool:
 
 On success, the command returns the details of the newly created work pool, which can then be used to start agents that poll this pool for work or perform additional configuration of the pool.
 
+
+#### Base Job Configuration <span class="badge beta"></span>
```

Comment:
Once you get down to H4 it can be pretty small, might want to make sure this renders well.